examples/intkey_go/src/sawtooth_intkey_client/utils.go

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
We try to void having util files. Instead, I suggest moving these functions to main.

Reply:
Ok, moved